account_credit_control/wizard/credit_control_policy_changer.py

```diff
@@ -52,18 +55,17 @@ def _get_default_lines(self):
         context = self.env.context
         active_ids = context.get('active_ids')
         invoice_obj = self.env['account.invoice']
-        move_line_obj = self.env['account.move.line']
         if not active_ids:
             return False
-        selected_lines = move_line_obj.browse()
+        selected_lines = self.env['account.move.line']
         for invoice in invoice_obj.browse(active_ids):
             if invoice.type in ('in_invoice', 'in_refund', 'out_refund'):
-                raise api.Warning(_('Please use wizard on customer invoices'))
+                raise Warning(_('Please use wizard on customer invoices'))
 
             domain = [('account_id', '=', invoice.account_id.id),
                       ('move_id', '=', invoice.move_id.id),
                       ('reconcile_id', '=', False)]
```

Comment:
Correct field, as I said above, is "reconciled". 

Field "full_reconcile_id" is a Many2one field for store reconcile_id when reconcile is full. This field is created by module "account_full_reconcile", that is not in dependencies.